client/src/pages/CustomerProfilePage.js

```diff
@@ -35,25 +35,31 @@ const PageContainer = styled.div`
 
   nav span {
     margin-right: ${layout.spacing(4)};
-    background: white;
+    
   }
 
 `;
+const useStyles = makeStyles({
```

Comment:
If you are applying styles to PageContainer with styled-components, you probably don't need to be using makeStyles here